backend/app/services/llm_dispatch.py

```
@@ -0,0 +1,41 @@
+"""
+Canonical producer for enqueueing LLM tasks (API-equivalent entrypoint).
+"""
+from __future__ import annotations
+
+from typing import Dict
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)